doc/source/admin/scaling.md

```diff
@@ -237,160 +154,116 @@ Handlers (as well as assignment methods) are not configurable when using **uWSGI
 
 ## Configuration
 
-### uWSGI
-
-Although this document goes in to significant detail about uWSGI configuration, many more options are available, as well
-as additional documentation about options described here. Consult the uWSGI documentation for more:
+### Gunicorn
 
-* [Configuring uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/Configuration.html)
-* [uWSGI Options](https://uwsgi-docs.readthedocs.io/en/latest/Options.html)
-* [Quickstart for Python/WSGI applications](https://uwsgi-docs.readthedocs.io/en/latest/WSGIquickstart.html)
+We will only outline a few of Gunicorn's options, consult the [Gunicorn documentation](https://docs.gunicorn.org/en/latest/settings.html) for more.
 
-Configuration is performed in the `uwsgi` section of `galaxy.yml`. You will find that the default, if copied from
-`galaxy.yml.sample`, is commented out. The default configuration options are provided to uWSGI on the command line by
-Galaxy's `run.sh` script.
 
-Galaxy releases prior to 18.01 (or upgraded-to-18.01+ servers which have not migrated their configuration to the YAML
-format) used an INI-format configuration file, `galaxy.ini`.
+Note that by default Galaxy will use [gravity](https://github.com/galaxyproject/gravity/) to create a [supervisor](http://supervisord.org/) configuration that
```

Comment:
Where is the supervisor configuration file created?

Reply:
You mean the command or the filesystem location ?

Reply:
Not sure how this works, I supposed that gravity creates a `supervisord.conf` file somewhere?

Reply:
Ah, yes, that's in gravity's state_dir, by default in `database/gravity/supervisor`